src/main/fc/config.c

```diff
@@ -165,7 +166,7 @@ static void validateAndFixConfig(void)
 
     if (
 #if defined(USE_GPS)
-        !findSerialPortConfig(FUNCTION_GPS) &&
+        !findSerialPortConfig(FUNCTION_GPS) && gpsConfig()->provider != GPS_MSP &&
```

Comment:
The test should be inverted - do not even look for port when GPS_MSP

Reply:
OK, please see my comment to @mikeller above.